src/rendering/RenderUtil.cc

```diff
@@ -2272,6 +2349,107 @@ void RenderUtil::ViewWireframes(const Entity &_entity)
     }
   }
 
+  return links;
+}
+
+/////////////////////////////////////////////////
+void RenderUtil::HideWireboxes(const Entity &_entity)
+{
+  if (this->dataPtr->wireBoxes.find(_entity)
+        != this->dataPtr->wireBoxes.end())
+  {
+    ignition::rendering::WireBoxPtr wireBox =
+      this->dataPtr->wireBoxes[_entity];
+    auto visParent = wireBox->Parent();
+    if (visParent)
+      visParent->SetVisible(false);
+  }
+}
+
+/////////////////////////////////////////////////
+void RenderUtil::ViewTransparent(const Entity &_entity)
+{
+  std::vector<Entity> visEntities;
+
+  if (this->dataPtr->linkToVisualEntities.find(_entity) !=
+      this->dataPtr->linkToVisualEntities.end())
+  {
+    visEntities = this->dataPtr->linkToVisualEntities[_entity];
+  }
+
+  // Find all existing child links for this entity
+  std::vector<Entity> links = this->FindChildLinks(_entity);
+
+  for (const auto &link : links)
```

Comment:
wrap this block with `{ ..}` brackets

Reply:
[`7f352fc`](https://github.com/ignitionrobotics/ign-gazebo/pull/878/commits/7f352fc39c453029cc2bc915135ca07c296f201c)